docs/UserGuide.adoc

```diff
@@ -14,7 +14,13 @@ endif::[]
 
 By: `Team SE-EDU`      Since: `Jun 2016`      Licence: `MIT`
 
-== Quick Start
+== Welcome to CYNC
+CYNC is an open-source software that helps small businesses to track and nurture relationships with customers. +
+Derived from the word " Sync ", CYNC aims to keep customers and businesses in sync. CYNC ensures that customers are businesses' main priority
+by storing customers' information and tracking their activities. CYNC also focuses on efficiency thus its
+Command Line Interface (CLI) allows users to add or retrieve customer's data quickly.
```

Comment:
> CYNC also focuses on efficiency thus its Command Line Interface (CLI) allows users to add or retrieve customer's data quickly.

Doesn't sound very logical, I think better phrased as something like 'CYNC's CLI enables users who prefer typing over clicking with the mouse to be more efficient. This is accomplished through various tools such as search prediction and blah blah.., which users to modify or retrieve customers' data quickly.'